google/generativeai/files.py

```diff
@@ -52,14 +55,20 @@ def upload_file(
     return file_types.File(response)
 
 
-def list_files(page_size=50) -> Iterable[file_types.File]:
+def list_files(max_results=50) -> Iterable[file_types.File]:
```

Comment:
The underlying GAPIC library handles paging automatically, and we return an  iterator. I'm not sure I see the goal of "max_results", which is unused.

```suggestion
def list_files(page_size=50) -> Iterable[file_types.File]:
```